### PR TITLE
Olegmikadze

### PR DESCRIPTION
**Pull request by olegmikadze**
Added "Forgot password " module window

### CODE REVIEW

File: front-end/src/components/Login/Forgot/index.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+{/*
+                <Link to="/forgot" className="forgot">Forgot password?</Link>
+*/}
+                <div className="forgot__dialog">
+                    <span className="forgot__close">&times;</span>
+                    <div className="forgot__wrapper">
+                        <form action="#" className="forgot__form">
+                            <label htmlFor="username">Enter email:</label>
+                            <input type="text" name="" id="username" placeholder="E-mail" autoComplete="off"/>
+                            {/*<label htmlFor="password">Password</label>
```
Comment (Thealoner) [inline front-end/src/components/Login/Forgot/index.js]: Commented code

File: front-end/src/App.js [hunk 2]
```diff
@@ -14,6 +15,8 @@ class App extends Component {
         <div className="app">
           <Header/>
           <Login/>
+          <Forgot/>
+          {/*<Route exact path="/forgot" component="forgot"/>*/}
```
Comment (Thealoner) [inline front-end/src/App.js]: It is not a good practice to commit commented code, unless you have a very good reason to do that. You should either remove the code completely if you no longer need it, or remove the comments and keep the code.

File: front-end/src/components/Login/Forgot/index.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+            $('.login').fadeIn();
+*/
+        });
+    };
+
+    render(){
+        return (
+
+            <div className="forgot">
+{/*
```
Comment (Thealoner) [inline front-end/src/components/Login/Forgot/index.js]: Commented code

Comment (olegmikadze) [reply front-end/src/components/Login/Forgot/index.js]: @Thealoner  Ok, now how i can change this if i already commited it? Delete this request and create new one?

File: front-end/src/components/Login/Forgot/index.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
+import React, {Component} from 'react';
+import './index.scss';
+import $ from 'jquery';
+
+class Forgot extends Component{
+
+    componentDidMount = () => {
+        $('.forgot__close').on('click', function () {
```
Comment (Thealoner) [inline front-end/src/components/Login/Forgot/index.js]: Please check tab settings in your IDE. Tab should be 2 spaces, not 4, according to the ESLint settings.
image

You should also configure ESLint plugin to enforce the rules in .eslint. Run 'npm install' first, I've added needed dependencies for ESlint to the package.json on this branch
eslint-config

File: front-end/src/components/Login/Forgot/index.scss [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+      width: 100%;
+      height: 30px;
+      margin: 10px 0 20px;
+      border: none;
+      outline: none;
+    }
+
+    & input[type="text"],
+    & input[type="password"],
+    & input[type="confirm_password"]{
```
Comment (YaroslavNikiforov) [inline front-end/src/components/Login/Forgot/index.scss]: input[type="confirm_password"] - ????????

Comment (YaroslavNikiforov) [reply front-end/src/components/Login/Forgot/index.scss]: it's a wrong input type

File: front-end/src/components/Login/Forgot/index.js (hunk 3 above)
Comment (Thealoner) [reply front-end/src/components/Login/Forgot/index.js]: Just edit/remove the lines that have the comments and commit the changes.

**Pull request closed without merging**